test/CMakeLists.txt

```diff
@@ -146,6 +146,13 @@ set(libyul_sources
 )
 detect_stray_source_files("${libyul_sources}" "libyul/")
 
+set(solc_sources
+    solc/CommandLineParser.cpp
+    ../solc/CommandLineParser.h
+    ../solc/CommandLineParser.cpp
```

Comment:
To avoid double-compilation you could also `add_library(solccli ...)` and use that in both places. I think that is cleaner than this cross-referencing of source files here. What do you think?

Reply:
I'll try.

Reply:
Thanks for the suggestion. It also solves a similar problem I had with yul-phaser. I created a separate PR for it (#11571) and rebased this one on top.